designer/client/src/components/ProcessComments.tsx

```diff
@@ -59,6 +60,14 @@ function ProcessComments(): JSX.Element {
 
     return (
         <div className="process-comments">
+            <button
```

Comment:
It seems like somethig temporary, can we remove it?

Reply:
oh, sorry My misstake

Reply:
done